src/libPMacc/include/traits/GetUniqueTypeId.hpp

```diff
@@ -24,9 +24,9 @@
 #pragma once
 
 #include "types.h"
-#include <iostream>
-#include <cassert>
-
+#include <sstream>
+#include <string>
+#include <exception>
```

Comment:
While testing this header in isolation I found that ```std::runtime_error``` is defined in ```<stdexcept>``` and not in ```<exception>```.
Could you additionally add ```#include <climits>``` for ```CHAR_BIT```?